configs/warnings.tslint.json

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
+    "deprecation": true,
```

Comment:
hello, I don't see how this change it is related to this feature ?

Reply:
I've deprecated some APIs and want other developers to notice it.

Reply:
my point is why deprecated notice is not discussed first
and why it should be deprecated